src/module/actor/entity.js

```diff
@@ -437,49 +437,50 @@ export default class OseActor extends Actor {
   rollAttack(attData, options = {}) {
     const data = this.system;
 
-    const rollParts = ["1d20"];
-    const dmgParts = [];
-    let label = game.i18n.format("OSE.roll.attacks", {
-      name: this.name,
-    });
-    if (attData.item) {
-      label = game.i18n.format("OSE.roll.attacksWith", {
-        name: attData.item.name,
-      });
-      dmgParts.push(attData.item.system.damage);
-    } else {
-      dmgParts.push("1d6");
-    }
+    const label = attData.item
+      ? game.i18n.format("OSE.roll.attacksWith", {
+          name: attData.item.name,
+        })
+      : game.i18n.format("OSE.roll.attacks", {
+          name: this.name,
+        });
+
+    const dmgParts = [
+      // Weapon damage roll value
+      attData.item?.system?.damage ?? "1d6",
+      // Weapon damage bonus
+      attData.item?.system?.bonus,
+    ].reduce((acc, curr) => {
```

Comment:
The reducer here is the same as what's on line 475. We could make that an arrow function constant so we only have to write it once.